server/service/src/processors/transfer/invoice/create_inbound_invoice.rs

```diff
@@ -153,7 +153,7 @@ fn generate_inbound_invoice(
     r#type: InboundInvoiceType,
 ) -> Result<InvoiceRow, RepositoryError> {
     let store_id = record_for_processing.other_party_store_id.clone();
-    let name_link_id = outbound_invoice.store_row.name_id.clone();
+    let name_link_id = outbound_invoice.store_row.name_link_id.clone();
```

Comment:
Nit: the name_link_id is only used once, i.e. this variable can be inlined where it is used which makes it easier to see that then name_link_id is used correctly...

However, think even better would be to use `outbound_invoice.name_row.id`. This would further reduce the useage of `name_link_id`

Reply:
Also when names are merged... doesn't the id stay as the kept name id since we're using id = name.id?

Reply:
Good point, you need to get the store name_id.

> Also when names are merged... doesn't the id stay as the kept name id since we're using id = name.id?

Yes, but the records could use the deleted name_link_id. It would still work, but it would be cleaner to use the name_link_id of the current name_id.